package.json

```diff
@@ -30,7 +30,7 @@
     "stylelint": "^10.0.0"
   },
   "peerDependencies": {
-    "stylelint": "^8.3.0 || ^9.0.0 || ^10.0.0"
+    "stylelint": ">=10.1.0"
```

Comment:
This will match all versions (including major) older or equal than 10.1.0.

Reply:
Did you mean _newer_ or equal than 10.1.0?

Reply:
Correct :)